performance.lic

```diff
@@ -0,0 +1,156 @@
+custom_require.call(%w(common common-travel))
+  
+class Performance
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    no_pause_all
+    startTime = Time.new.to_i
+
+    @settings = get_settings
+    @song_list = get_data('perform').perform_options
+    UserVars.song = @song_list.first.first unless UserVars.song
+    fput('release ecry') if DRStats.bard?
+
+    instrument = @settings.instrument
+
+    hand_armor = @settings.hand_armor
+    if instrument != nil
+      bput("get my #{instrument}","You get")
+      bput("remove my #{hand_armor}","You pull off","Remove what")
+      bput("stow my #{hand_armor}","You put","Stow what")
+    end
+    
+    if play_song
+      loop do 
+        runTime = Time.new.to_i - startTime
+        break if runTime > 600
+        break if DRSkill.getxp('Performance') > 30
+        line = get
+        pause 0.05 unless line
+        case line
+        when /You begin (.*)/, /You continue (.*)/, /You find yourself relaxing (.*)/, /You concentrate on your sense of rhythm (.*)/, /You continue (.*)/
+          if DRSkill.getxp('Performance') > 30
+            fput('stop play')
+            exit
+          end
+        when /You finish (.*)/
+          if DRSkill.getxp('Performance') <= 30
+            play_song
+          else
+            exit
+          end
+        when /Perhaps you should find somewhere drier/
+          exit
+        end
+      end
+    end
+     
+    if instrument != nil
+      bput("stow my #{instrument}","You put")
```

Comment:
💡 By convention, strings that are 'simple' (no apostrophe, no interpolation) should be single-quoted like `'You put'`. Rubocop will fix this for us.